### PR TITLE
Update Distaria (Colors and Insta)

### PR DESCRIPTION
**Pull request by 1combit**
## Type of change

- [ ] Add new directory for a server
- [ x ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/distaria/manifest.json [hunk 1]
```diff
@@ -10,8 +10,13 @@
       "twitter": "DistariaNetwork",
       "discord": "https://discord.gg/asYBeq8QYD",
       "teamspeak": "distaria.net",
+      "instagram": "distarianet",
       "web_support": "https://distaria.net/forum/index.php?board/4-support/"
     },
+    "brand": {
+      "text": "#306fdb",
+      "background": "#306fdb"
```
Comment (JakobBruening) [inline minecraft_servers/distaria/manifest.json:18]: I'm sure that won't look so good 😬
It's like putting white text on a white background. 😅
I guess you have to change the text color.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/distaria/manifest.json (hunk 1 above)
Comment (1combit) [reply minecraft_servers/distaria/manifest.json:18]: oh i now i check what the background color is

**Comment by 1combit** [comment]
So editet

**Review by JakobBruening — APPROVED** [review]
Thank you!

**Pull request merged**